src/dynamics/ads_player/ads_player.html

```diff
@@ -64,7 +64,7 @@
     ba-userhadplayerinteraction="{{userhadplayerinteraction}}"
     ba-currenttime={{=currenttime}}
     ba-hideoninactivity={{hideoninactivity}}
-    ba-tooltips="{{tooltips}}"
+    ba-tooltips="{{isMobile ? '' : tooltips}}"
```

Comment:
The tooltips rendering on mobile was causing for duplicate views:
<img width="683" alt="Screenshot 2024-07-30 at 1 52 36 PM" src="https://github.com/user-attachments/assets/0ba79895-9d24-4657-abcc-305618799874">
On top of that, for mobile, you must click on Google's `Learn More` specifically to be redirected, so the tooltip message from us was misleading.